setup.py

```diff
@@ -21,8 +21,10 @@ def read(filename):
     'requests',
     'feedparser',
     'pytz',
+    'google-api-python-client',
```

Comment:
But the 2 commits ([the one adding the support](https://github.com/Doist/powerapp/commit/8b8e629bbef3e78dc6ffa80823e7db87f5269402#diff-2eeaed663bd0d25b7e608891384b7298) and [the merge with master](https://github.com/Doist/powerapp/commit/62803535ff98aad2e7620b9b3b3bb73744cfac95#diff-2eeaed663bd0d25b7e608891384b7298) don't contain and line like that, that is strange...

Was that like removed by you, and readded by the merge?